doc/api/deprecations.md

```diff
@@ -0,0 +1,567 @@
+# Deprecated APIs
+
+Node.js will deprecate APIs when either: (a) use of the API is considered to be
+unsafe, (b) an improved alternative API has been made available, or (c)
+breaking changes to the API are expected in a future major release.
+
+Node.js utilizes two kinds of Deprecations:
+
+* Documentation-only
+* Runtime
+
+A Documentation-only deprecation is one that is expressed only within the
+Node.js API docs. These generate no side-effects while running a Node.js
+process.
+
+A Runtime deprecation will, by default, generate a process warning that will
+be printed to `stderr` the first time the deprecated API is used within a
+Node.js process. When the `--throw-deprecation` command-line flag is used,
+a Runtime deprecation will cause an error to be thrown.
+
+## Un-deprecation
+
+From time-to-time, the Node.js Core Technical Committee (CTC) may decide to
+reverse the decision to deprecate an API. Such action may happen in either a
```

Comment:
I’d drop the mention of the CTC. It’s an implementation detail and it’s ultimately irrelevant who makes these decisions; how about just switching to passive voice, i.e. `From time to time, the deprecation of an API may be reversed.`?